src/aadeshJun20/SwapNumbers.java

```diff
@@ -0,0 +1,21 @@
+package aadeshJun20;
+
+public class SwapNumbers {
+
+	void displaySwapping(int num1, int num2) {
+		System.out.println("Input number 1= " + num1 + ", number 2 = " + num2);
+		num1 = num1 + num2;
+		num2 = num1 - num2;
+		num1 = num1 - num2;
+		
```

Comment:
Please remove extra space here. and line 13 and 14 as well

Reply:
changes incorporated